src/App.tsx

```diff
@@ -1,61 +1,36 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { useState } from 'react';
+import todosFromServer from './api/todos';
```

Comment:
[CHECKLIST ITEM 5] Prepare data in one place (App component) and pass it to child components. Importing 'todosFromServer' directly in the App component is correct, but you should also prepare the 'neededTodos' in the App component and pass it to the child components instead of calling 'getUserById' in the TodoList or other child components.